app/artifact-cas/cmd/main.go

```diff
@@ -64,16 +62,6 @@ type app struct {
 	backend.Providers
 }
 
-func loadCASBackendProviders(creader credentials.Reader) backend.Providers {
```

Comment:
this has been moved to a common pkg `/internal/blobmanager/loader` since it is being used by both the CAS and the controlplane